pkg/controller/instance/instance_controller.go

```diff
@@ -147,14 +139,16 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 				}
 				// Not currently doing anything for Dependency changes.
 			} else {
-				log.Println("InstanceController: Old and new spec matched...")
-				planName = "deploy"
+				if new.Status.ActivePlan.Name == "" {
+					log.Printf("InstanceController: Old and new spec matched...\n %+v ?= %+v\n", old.Spec, new.Spec)
+					planName = "deploy"
+					ok = true
```

Comment:
Adding this line fixes a bug that is not related to using `patches` to update things. I would pull it out to it's own PR with a nice description of what has been fixed.